src/plugins/georeferencer/qgsimagewarper.cpp

```diff
@@ -97,7 +97,12 @@ bool QgsImageWarper::createDestinationDataset( const QString &outputName, GDALDa
   if ( crs.isValid() )
   {
     OGRSpatialReference oTargetSRS;
+    //oTargetSRS.importFromProj4( crs.toProj().toLatin1().data() );
+#if PROJ_VERSION_MAJOR>=6
+    oTargetSRS.importFromWkt( crs.toWkt().toLatin1().data() );
```

Comment:
Better export in wkt 2018 format so it's not lossy

Reply:
Done.

Reply:
toUtf8() would likely be more appropriate. proj.db content is encoded in UTF-8

Reply:
@rouault , roger that.